library/system/src/lib/yast2/fs_snapshot.rb

```diff
@@ -139,6 +152,36 @@ def self.create(snapshot_type, description, previous = nil)
     end
     private_class_method :create
 
+    # detects if module runs in initial stage before scr is switched to target system
```

Comment:
I cannot found link, but I read somewhere that it is recommended to use it only when it make sense, so having param or return that just copy general description method is quite useless for me. Is there reason for it? E.g. in Store class I use it where it make sense ( like raising exception, or non-obvious return type )